doc/userguide/3rd-party-integration/symantec-sslv.rst

```diff
@@ -0,0 +1,90 @@
+Symantec SSL Visibility (BlueCoat)
+==================================
+
+As Suricata itself cannot decrypt SSL/TLS traffic, some organizations use
+a decryption product to handle this. In this document some advice about
```

Comment:
maybe just me but this sentence seem to be missing something like "In this document we will give some advice" or so?